resources/lib/ui/embed_extractor.py

```diff
@@ -67,8 +79,11 @@ def __9anime_extract_direct(refer_url, grabInfo):
 def __extract_9anime(url, page_content):
     episode_id = url.rsplit('/', 1)[1]
     domain = urlparse.urlparse(url).netloc
-    url = "http://%s/ajax/episode/info?id=%s&update=0" % (domain, episode_id)
-    grabInfo = json.loads(http.send_request(url).text)
+    scheme = urlparse.urlparse(url).scheme
+    url = "%s://%s/ajax/episode/info?id=%s&update=0" % (scheme, domain, episode_id)
```

Comment:
i guess they will move to the token for all so better be safe and add it to all requests if possible